AMW_angular/io/src/app/resources/resources-page.component.html

```diff
@@ -2,30 +2,38 @@
 <app-page>
   <div class="page-title">Resources</div>
   <div class="page-content">
-    @if (permissions().canViewResourceTypes) {
-    <nav class="nav flex-column">
-      @for (resourceType of predefinedResourceTypes(); track resourceType.name) {
-      <a>{{ resourceType.name }}</a>
-      }
-    </nav>
-    <nav class="nav flex-column">
-      @for (resourceType of rootResourceTypes(); track resourceType.name) {
-      <a (click)="toggleChildren(resourceType)">
-        {{ resourceType.name }}
-        @if (resourceType.hasChildren) {
-        <span>{{ expandedResourceTypeId === resourceType.id ? '-' : '+' }}</span>
+    <div class="row">
+      <div class="col">
+        @if (permissions().canViewResourceTypes) {
+          <nav class="nav flex-column">
+            @for (resourceType of predefinedResourceTypes(); track resourceType.name) {
+              <a (click)="toggleChildrenAndOrLoadResourcesList(resourceType)">{{ resourceType.name }}</a>
+            }
+          </nav>
```

Comment:
maybe a little space between predefined and rootResources? dont now

Reply:
What do you think about that? I have improved the user interface of the resource pages (especially resource types) in an other PR: https://github.com/liimaorg/liima/pull/803. If it's ok, I will make this adjustment there.

![image](https://github.com/user-attachments/assets/a45f5ffe-1a5d-4c4c-a51f-705ee9561f54)